README.md

```diff
@@ -1,10 +1,13 @@
+
 ## Docker for Visual Studio Code
 
 [![Version](https://vsmarketplacebadge.apphb.com/version/ms-azuretools.vscode-docker.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) [![Installs](https://vsmarketplacebadge.apphb.com/installs-short/ms-azuretools.vscode-docker.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) [![Build Status](https://dev.azure.com/ms-azuretools/AzCode/_apis/build/status/Nightly/vscode-docker-nightly-2?branchName=master)](https://dev.azure.com/ms-azuretools/AzCode/_build/latest?definitionId=22&branchName=master)
```

Comment:
I just realized you can put these badges next to the title e.g. [how Serilog does it](https://github.com/serilog/serilog/blob/dev/README.md) That will save some screen real estate

Reply:
I like it!

Reply:
Agree, looks good